reactComponents/src/searchbar/tokenSearch/Chip.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { useContext, FC } from 'react';
+import styled from 'styled-components';
+import TokenSearchContext from '../Context/TokenSearch';
+
+import { Logo } from './Logo'
+import CheckedIcon from '../icons/checked'
+import UnCheckedIcon from '../icons/unchecked';
+
+const StyledChip = styled.div`
+    ${({ styles }) => `
+        > input {
+          display: none;
+        }
+
+        > input + label {
+          -webkit-transition: all 500ms ease;
+          transition: all 500ms ease;    
+          cursor: pointer;    
+          display: grid;
+          align-items: center;
+              
+          -moz-user-select: -moz-none;
```

Comment:
Does Rome Terminal use autoprefixer? These shouldn't have to be written by hand.